packages/sol-cov/src/trace.ts

```diff
@@ -95,10 +102,15 @@ export function getTracesByContractAddress(structLogs: StructLog[], startAddress
         }
     }
     if (callStack.length !== 0) {
-        throw new Error('Malformed trace. Call stack non empty at the end');
+        logUtils.warn('Malformed trace. Call stack non empty at the end');
```

Comment:
Do we want to continue after finding a malformed trace?

Reply:
Yes. Some traces run out of gas and look truncated. Also - this tool should be transparent. It shouldn't break/stop your tests. I think - printing a warning is enough.